automation/publish.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash -xe
+
+# This script publishes macvtap-cni by default at quay.io/kubevirt
+# organization.
+# To publish elsewhere export the following env vars
+# IMAGE_REGISTRY
+# IMAGE_REPO
+# IMAGE_TAG
+# To run it just do proper docker login and automation/publish.sh
+
+source automation/check-patch.setup.sh
+cd ${TMP_PROJECT_PATH}
+make docker-build
+make docker-tag-latest
+make docker-push
```

Comment:
AFAIK, this would push only the :$IMAGE_TAG one, but not latest. You need to call `push` on every tag you want to publish.

Reply:
Hm, I see.

Done.